src/posts/action.md

```diff
@@ -4,61 +4,32 @@ description: Ways to help out with pausing AGI development.
-  - Having some printed one-page flyers on hand, or easily available to show and share from your phone, can be a quick and useful resource for faqs, talking points, expert quotes and opinions, online resources, etc.
-- **Create** [articles](/learn#articles), [videos](/learn#videos) or [memes](https://twitter.com/AISafetyMemes). Collaborate with others in the [Discord server](https://discord.gg/2XXWXvErfA) (in the #projects channel).
 
 ## Inform politicians
 
-- [**Send an email to a politician**](/email-builder): emails are completely overpowered. Do it.
-- [**Lobby**](/lobby-tips): convince your government to work towards a pause and prepare for the [summit](/summit).
-- [**Protest**](/protests): join [one of the protests](/protests) or [organize one yourself](/organizing-a-protest).
-- **Sign petitions**:
-  - [International AI Treaty](https://aitreaty.org)
-  - [Ban Superintelligence](https://chng.it/Djjfj2Gmpk)
-  - [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df)
-  - Or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
+- [**Send an email to a politician**](/email-builder): emails are completely overpowered. Try to convince your government to work towards a pause and prepare for the [summit](/summit). Here are some [**lobby tips**](/lobby-tips).
+- **Protest**: join [one of the protests](/protests) or [organize one yourself](/organizing-a-protest).
+- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
 
 ## Join the movement
 
-- **[Join PauseAI](/join)** and help us grow.
-- **Join our [Discord server](https://discord.gg/T3YrWUJsJ5)**, where our community is most active. We have a **#projects** channel where people are working on campaigns, videos, images, apps and more.
-- **Join one of our [teams](/teams)** and collaborate with other motivated volunteers.
-- **Meet** other people in a [PauseAI community near you](/communities). If there is no community near you, [starting one](/local-organizing).
+- Join or create a [local community](/communities) or [event](/events). 
+- Join the PauseAI [Discord](https://discord.gg/2XXWXvErfA), where most of the collaboration happens.
+- If you want to volunteer, fill out [this form](https://airtable.com/embed/appWPTGqZmUcs3NWu/pagoxRuCai4OYJEHt/form).
```

Comment:
This is a radical change, since pretty much the entire website converges to `/join`

Reply:
How do a radical change to said key page? It just adds the `/events` link though, doesn't it?

Reply:
> How so a radical change to said key page? It just adds the `/events` link, doesn't it?

Nono, that's the change to the /join page. The change Joep pointed up to is in the /action page.

It's better to reduce the amount of clicks, no? If not why don't we just put "[**Join us**](/join)"?
The present section is really repetitive and confusing.